lms/static/scripts/frontend_apps/config.ts

```diff
@@ -83,6 +83,7 @@ export type FilePickerConfig = {
   formAction: string;
   formFields: Record<string, string>;
   promptForTitle: boolean;
+  autoGradingEnabled: boolean;
```

Comment:
Same as in the feature-flag PR: https://github.com/hypothesis/lms/pull/6633/files#diff-bff29b887d9d4de6137242d39e8b3c6f076825c1e4baf2580cf4ac77172fcf78